src/web/js/trove/world.js

```diff
@@ -575,10 +545,50 @@
       return rawJsworld.stop_when(worldFunction);
     };
 
+    var LastPicture = function(handler) {
+      WorldConfigOption.call(this, 'last-image');
+      this.handler = handler;
+    };
+
+    LastPicture.prototype = Object.create(WorldConfigOption.prototype);
+
+    LastPicture.prototype.toRawHandler = function(toplevelNode) {
+      var that = this;
+      var reusableCanvas;
+      var lastPictureFunction = function() {
```

Comment:
Added PR #191 that puts in all these suggestions and supersedes this PR. (Used new PR so as keep track of changed `horizon`.)